MoviesMVP/MoviesMVPTests/NetworkServiceTests.swift

```diff
@@ -0,0 +1,123 @@
+// NetworkServiceTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import XCTest
+
+@testable import MoviesMVP
+
+/// Network service tests
+final class NetworkServiceTests: XCTestCase {
+    // MARK: - Public properties
+  
+    var networkService: NetworkServiceProtocol?
+    var movies: [Movie]?
+    var movieDetail: MovieDetail?
+    var cast: [Cast]?
+
+   // MARK: - Public methods
+  
+    override func setUp() {
+        networkService = MockNetworkService()
+    }
+
+    override func tearDown() {
+        networkService = nil
+    }
+
+    func testSuccessMovies() {
+        movies = [Movie]()
+        networkService = MockNetworkService(movies: movies)
+        var catchMovies: [Movie]?
+
+        networkService?.fetchMovies(categoryOfMovies: "popular", page: 1) { result in
+            switch result {
+            case let .success(movies):
+                catchMovies = movies
+            case .failure:
+                break
+            }
+        }
+        XCTAssertEqual(catchMovies?.count, 1)
+        movies = nil
+    }
+
+    func testFailureMovies() {
+        networkService = MockNetworkService(movies: movies)
+        var catchError: Error?
+        networkService?.fetchMovies(categoryOfMovies: "popular", page: 1) { result in
```

Comment:
```suggestion
        networkService?.fetchMovies(categoryOfMovies: "popular", page: 1) { result in
```
в константы. и page -- тоже